docs/upgrade-guide-1.6-to-1.7.ipynb

```diff
@@ -65,11 +80,11 @@
     ]
    },
    "source": [
-    "### Update default 'admin' profile to prevent its deletion\n",
+    "### Update default `admin` profile to prevent its deletion\n",
```

Comment:
I wonder if, since it seems like Juju sometimes reapplies these labels, instead of having this admin profile workaround in "before you upgrade" it would fit better as a note down in "upgrade charms".  That way when people apply it, they do so right before actually refreshing.

Or, maybe at least have a note down in Upgrade Charms that says "juju may re-apply these labels - if you want to preserve the admin profile, check using `some command to test labels` and optionally re-remove before refreshing kubeflow-dashboard